projects/dynamic-form/src/lib/components/main-form/main-form.component.html

```diff
@@ -1,157 +1,202 @@
-<div>
-  <form [formGroup]="myForm">
-    <div *ngFor="let control of formJson">
-      <mat-form-field
-        *ngIf="control?.type === 'text' || control?.type === 'password'"
-        floatLabel="always"
+<form [formGroup]="myForm" [class]="classFlex">
```

Comment:
Why every contents in this file is shown as modified?
Is it a formatting issue, if yes please revert it